src/Client/Connection.cpp

```diff
@@ -288,6 +309,19 @@ void Connection::forceConnected(const ConnectionTimeouts & timeouts)
     }
 }
 
+#if USE_SSL
+void Connection::sendCluster()
```

Comment:
This function also sends salt. Could you rename it into `sendClusterNameAndSalt()` or something.

Reply:
By Cluster I mean "info for cluster", but I'm ok with naming it sendClusterNameAndSalt

Reply:
This is pending, I don't want to burn CI resources, so please let me know when you will finish (you can just add summary to your review, i.e. "Request changes" or similar)

Reply:
Have you catched anything else?